legal/geschaeftsordnung.md

```diff
@@ -0,0 +1,27 @@
+# Geschäftsordnung
```

Comment:
Transparenzanforderungen fehlen: Z.B. Protokoll der Vorstandstreffen? etc. was öffentlich / nicht öffentlich?

Nachvollziehbarkeit der internen Entscheidungen?

Reply:
1. Beschlüsse können nur auf Versammlungen bzw. Sitzungen (im Folgenden Sitzungen) des jeweils beschlussfassenden Organs des Vereins getroffen werden.
2. Sitzungen der Organe des Vereins sind zu protokollieren. Die Protokolle gliedern sich in einen öffentlichen und einen nichtöffentlichen Teil. Sie sind mit Datum und Uhrzeit der Sitzung zu versehen und jeweils fortlaufend zu nummerieren.
3. Nichtöffentliche Teile werden in einer eigenen Datei protokolliert und separat abgelegt, diese wird unter Nennung der sha1-Summe im öffentlichen Protokoll referenziert und somit zum untrennbaren Bestandteil des Protokolls.
4. Der öffentliche Teil ist von Protokolant*in und Versammlungsleitung (ggf. digital) zu signieren und auf der Webseite des Vereins zu veröffentlichen.